arm-compute/2017-03-30/swagger/runCommands.json

```diff
@@ -0,0 +1,303 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "RunCommandsClient",
+    "description": "The Run Commands Client.",
+    "version": "2017-03-30"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
+  ],
+  "produces": [
+    "application/json",
+    "text/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute/locations/{location}/runCommands": {
+      "get": {
+        "tags": [
+          "VirtualMachineRunCommands"
+        ],
+        "operationId": "VirtualMachineRunCommands_List",
+        "description": "Lists all available run commands for a subscription in a location.",
+        "parameters": [
+          {
+            "name": "location",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The location upon which run commands is queried.",
+            "pattern": "^[-\\w\\._]+$"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/RunCommandListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute/locations/{location}/runCommands/{commandId}": {
+      "get": {
+        "tags": [
+          "VirtualMachineRunCommands"
+        ],
+        "operationId": "VirtualMachineRunCommands_Get",
+        "description": "Gets specific run command for a subscription in a location.",
+        "parameters": [
+          {
+            "name": "location",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The location upon which run commands is queried.",
+            "pattern": "^[-\\w\\._]+$"
+          },
+          {
+            "name": "commandId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The command id."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/RunCommandDocument"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/runCommand": {
+      "post": {
```

Comment:
@koralski 
Please add a JSON file that describes this and add a reference here. The documentation for this can be found here: https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/x-ms-examples.md.